evaluation/analysis.py

```
@@ -10,161 +10,174 @@
-    format='svg'
-)
+    # get router barplots
+    router_alignment_results: dict[str, list[float]] = {model: [] for model in set(list(router_eval_df['model']))}
+    for _, router_eval_row in router_eval_df.iterrows():
+        router_alignment_results[router_eval_row['model']].append(router_eval_row['prompt_alignment'])
+    ROUTER_BARPLOT_ALIGNMENT = barplot_metric(
+        'prompt_alignment',
+        router_alignment_results
+    )
+    ROUTER_BARPLOT_ALIGNMENT.savefig(
+        str(PATH_EVALUATION_PLOTS / f'router_barplot_alignment.svg'),
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)